packages/safety-suite-api/src/api/dashboardSink/kpi.ts

```diff
@@ -0,0 +1,39 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiSuccessResponse} from '../../types';
+import {KpiQuery} from './types';
+
+export interface KpiGroup {
+  name: string;
+  fields: string[];
+  values: unknown[];
+  humanName: string;
+}
+
+export interface KpiMetadata {
+  query: any;
+  page: number;
+  groups: KpiGroup[];
+}
+
+export interface KpiResponse<Data = unknown> {
+  meta: KpiMetadata;
+  data: Record<string, Data[][]>;
+}
+
+export function getKpi<Data>(
+  kpi: string,
+  query: KpiQuery,
+  apiOptions: ApiOptions = {}
+): Request<ApiSuccessResponse<KpiResponse<Data>>> {
+  return runApi({
+    method: 'GET',
+    urlRoot: 'dashboardSinkUrlRoot',
+    route: `/api/kpi/${kpi}`,
+    apiOptions,
+    requestOptions: {
+      query
```

Comment:
The customer alias is a query string, but the rest I thought were request body values.  Hence why its a POST.  If Mason has implemented them as query strings then this is fine for now, but that's something we'll have to look into changing.